docs/java-rest/high-level/supported-apis.asciidoc

```diff
@@ -103,7 +104,11 @@ include::indices/get_alias.asciidoc[]
 include::indices/put_settings.asciidoc[]
 include::indices/get_settings.asciidoc[]
 include::indices/put_template.asciidoc[]
+<<<<<<< HEAD
+include::indices/validate_query.asciidoc[]
+=======
 include::indices/get_templates.asciidoc[]
+>>>>>>> 0bfd18cc8b6ab7e58d1d34c50ae7a3b441799761
```

Comment:
this may be why the docs don't build? :)

Reply:
Yes. Thanks. Seems I forgot to build the docs after merging...